<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>751</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 20px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
background-color: rgb(229, 229, 229);

</string>
   </property>
   <property name="placeholderText">
    <string>输入要搜索的事务信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1581</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>1551</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);
border:1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 20px;/*设置圆角的大小*/</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 20px;/*设置圆角的大小*/</string>
    </property>
    <property name="text">
     <string>添加闹钟</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 20px;/*设置圆角的大小*/</string>
    </property>
    <property name="text">
     <string>消息通知</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>130</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 20px;/*设置圆角的大小*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>271</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;/*设置边框的粗细，以及颜色*/
background-color: rgb(229, 229, 229);
border-radius: 20px;/*设置圆角的大小*/</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>143</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_xinming">
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>姓名:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_xuehao">
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>学号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ID">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>学院:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_college">
         <property name="text">
          <string>计算机学院</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>专业:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_major">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>计算机科学与技术</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>91</y>
     <width>751</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 229, 229);
border-radius: 20px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>tableWidget</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
